packages/supabase/package.json

```diff
@@ -7,7 +7,6 @@
   "main": "./src/index.ts",
   "types": "./src/index.ts",
   "scripts": {
-    "build": "bun run types && bun run schemas",
```

Comment:
Shouldn't we have this script available to generate the TS and schemas?

There are another that is to pull the DB and push into different environments at supabase, but I see they hasn't been added yet to `develop` branch.

Reply:
Gabo told me to remove it while we were fixing the building error, should i add it back?

Reply:
Ohh I see! I will add it back, no worries. I am near and I will trigger a vercel build just to check status

Reply:
UPDATE: It went through :+1: I will be merging with this new line

Reply:
Ohh amazing! Thank you very much Andler 🙏🏻 and also Tks for the other changes you did 🙏🏻